src/pages/publication/my_publications/MyPublications.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { FC, useState, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSelector, useDispatch } from 'react-redux';
+import { Button, CircularProgress, Card } from '@material-ui/core';
+import styled from 'styled-components';
+import { RootStore } from '../../../redux/reducers/rootReducer';
+import { Link as RouterLink } from 'react-router-dom';
+import { getMyPublications } from '../../../api/publicationApi';
+import { setNotification } from '../../../redux/actions/notificationActions';
+import { NotificationVariant } from '../../../types/notification.types';
+import { PublicationPreview, PublicationStatus } from '../../../types/publication.types';
+import PublicationList from './PublicationList';
+import TabButton from '../../../components/TabButton';
+
+const StyledContainer = styled.div`
+  display: block;
+  width: 100%;
+  margin: 0 2rem 2rem 2rem;
+`;
+
+const StyledButtonWrapper = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  padding: 0 1rem;
+  height: 3rem;
+`;
+
+const StyledTabsContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  padding-top: 2rem;
+  margin: 0 1.5rem;
+`;
+
+const StyledProgressContainer = styled.div`
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  padding: 2rem;
+`;
+
+const StyledCard = styled(Card)`
+  margin: 0 1.5rem;
+`;
+
+enum Tab {
```

Comment:
vi har stort sett brukt UPPERCASE for enums, men burde kanskje gå over til PascalCase? 
https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
tror kanksje vi har brukt det fordi vi har for det meste hatt strings? og da er det vel naturlig å ha uppercase? men ja, vi kan godt gå over til PascalCase ja. Ser ut som det er mest brukt 👍